demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c

```diff
@@ -921,8 +918,8 @@ static void prvWriteHexBytesToConsole( CK_SESSION_HANDLE xSession,
         if( 0 == ( ( ulIndex + 1 ) % BYTES_TO_DISPLAY_PER_ROW ) )
         {
             *pcNextChar = '\0';
-            vLoggingPrint( pcByteRow );
-            vLoggingPrint( "\r\n" );
+            print_log( pcByteRow );
```

Comment:
Looks like this is a gap in the #defines we provide in the reference. We should offer one for printing naked log lines, rather than requiring you to make these modifications.

Reply:
In the commit message, the print log issue is mentioned in the TODO list. The plan is to use the longing task in libraries/c_sdk, thus the gap can be dismissed. I will handle the issue in the next commit.